components/brave_wallet/browser/internal/polkadot/rust/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "sr25519"
+version = "1.0.0"
+edition = "2021"
+license = "MPL-2.0"
+
+[dependencies]
+brave_wallet = { version = "1" }

```

Comment:
do we need brave_wallet dependency here?

Reply:
It turned out this entire Cargo.toml was actually unnecessary, ha ha. I had deleted it and nothing failed to compile.